### PR TITLE
Add pysam 0.10

### PR DESCRIPTION
**Pull request by kyleabeauchamp**
* [ ] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by kyleabeauchamp** [comment]
Anyone seen this one before?  

```
ERROR bioconda_utils.build:build(129): STDOUT+STDERR: bash: cannot set terminal process group (-1): Inappropriate ioctl for device
bash: no job control in this shell
```

**Comment by bgruening** [comment]
@kyleabeauchamp you can ignore this. It's a limitation of the conda-forge Docker container.

**Comment by bgruening** [comment]
This seems to be the error: https://travis-ci.org/bioconda/bioconda-recipes/jobs/194293085#L3305

**Comment by kyleabeauchamp** [comment]
Yeah, this is one of those strange linker errors that pysam's build process always seems to uncover.  Not sure exactly what is going on, I know they did some cleaning up of the setup.py in pysam recently.

File: recipes/pysam/0.9.1.4/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+    number: 1
+    skip: False
+
+requirements:
+    build:
+        - gcc  # [linux]
+        - llvm # [osx]
+        - htslib >=1.3
+        - samtools >=1.3
```
Comment (yhoogstrate) [inline recipes/pysam/0.9.1.4/meta.yaml]: Is this dependency really needed?

**Pull request closed without merging**